recipes/bam2fasta/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: bam2fasta
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/b/bam2fasta/bam2fasta-1.0.1.tar.gz
```

If you set the URL as
```suggestion
  url: https://pypi.io/packages/source/b/bam2fasta/bam2fasta-{{ version }}.tar.gz
```
I think the `autobump` bot will be able to create an update PR when a new version is released to PyPI.